packages/core/src/index.ts

```diff
@@ -1156,6 +1191,14 @@ class Hopr extends EventEmitter {
     return result
   }
 
+  /**
+   * @param id the peer id of the account we want to check if it's allowed access to the network
+   * @returns true if allowed access
+   */
+  public async isAllowedAccess(id: PeerId): Promise<boolean> {
```

Comment:
Naming is unclear. Allowed access to what?

Reply:
lets change this into `isAllowedAccessToNetwork` ?